cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/local/LocalStorageClient.java

```diff
@@ -75,10 +83,22 @@ public synchronized void init() throws URISyntaxException, IOException {
               + LOCAL_TYPE.getValue(),
           e);
     }
-    this.fs = FileSystem.get(uri, new org.apache.hadoop.conf.Configuration());
-    Preconditions.checkArgument(
-        fs instanceof LocalFileSystem,
-        "Instantiation failed for LocalStorageClient, fileSystem is not a LocalFileSystem");
+    this.fs = FileSystem.get(uri, hadoopConfig);
+    assertLocalFileSystem(fs);
+  }
+
```

Comment:
Can we add a specific comment on the causality of this change? ie. Trino supplies a FilterFileSystem wrapped instance of LocalFileSystem

Reply:
updated!